php/symfony/config.yaml

```diff
@@ -1,14 +1,11 @@
 framework:
   website: symfony.com
-  version: 5.1
+  version: 5.2
 
 deps:
   - nginx
 
-environment:
-  APP_ENV: prod
-
 before_command:
   - mkdir -p var/log && chown www-data var/log
   - mkdir -p var/cache && chown www-data var/cache
-  - php bin/console cache:warmup
+  - php bin/console cache:warmup --no-debug
```

Comment:
I'm not sure we need a warmup here, this step is done with wrk 
https://github.com/the-benchmarker/web-frameworks/blob/f7f7984535d7dd43f0e034bdadc679ce67dc3fb4/lib/tasks/collect.rake#L36

Reply:
Thank you for the review. 

Is that line running a http request before we actually start measuring the responses? If so, yes, you might be correct if docker allows me to write to the filesystem. 

This line is best practice to always run before you ship your project to production. I'll prefer to keep it.

Reply:
We can keep it, yep. However, `symfony` is the only (or one of) framework that have a built-in console and a warm up command. The above line is doing the same think (i think), it makes several HTTP requests so a all frameworks are warn for the real measurement